docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -268,6 +268,89 @@ When the client authenticates with the API ML, the client receives the JWT token
 authentication. If z/OSMF is configured as the authentication provider and the client already received a JWT token produced
 by z/OSMF, it is possible to reuse this token within API ML for authentication.  
 
+### Authentication parameters
+Parameters are specified in the onboarding enablers.
+
+Authentication parameters enables a service to accept the Zowe JWT or client certificate. The API Gateway translates the authentication token to an authentication method supported by a service.
+
+The following example shows the parameters that define the service authentication method:
+
+**Example:**
+
+```yaml
+authentication:
+    scheme: httpBasicPassTicket
+    applid: ZOWEAPPL
+```
+where:
+
+* **authentication.scheme**
+
+  specifies a service authentication scheme. Any valid headers or `X-Zowe-Auth-Failure` error headers will be set and passed to southbound services. In addition, any `X-Zowe-Auth-Failure` error headers coming from the northbound service will also be passed to the southbound services without setting the valid headers. The `X-Zowe-Auth-Failure` error header contains details about the error and suggest potential actions.
```

Comment:
Nitpick: Missing `This value` or something similar at the start.

Reply:
addressed

Reply:
Reworded to `is the value that specifies`.. at the beginning.